### DISCUSSION TITLE
Smart contract deployment example using Web3

### DISCUSSION THREAD

**Discussion by mnikonov** [nervosnetwork/godwoken#1138] [General]
Can you point me to documentation or better example where would be described how to deploy sol smart contract using web3 (^4.9.0)? All examples that I have found so far not working with latest versions of packages 

As well as I haven't found any information why [polyjuice-provider] and [godwoken-web3] repos was archived... It makes me feel like web3 support was ended?

**Comment by Flouse** [comment]
- duplicated with https://github.com/godwokenrises/godwoken/issues/1139
